storm-core/pom.xml

```diff
@@ -184,6 +184,10 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
```

Comment:
nit: can we make this test here too, just so it is clear that it is a test only API?

Reply:
Responded in the other PR